### DISCUSSION TITLE
How to fix this bug?

### DISCUSSION THREAD

**Discussion by nissansz** [fastapi/fastapi#11045] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

Python3.8.8\lib\site-packages\fastapi\params.py", line 4, in <module>
    from pydantic.fields import FieldInfo, Undefined
ImportError: cannot import name 'Undefined' from 'pydantic.fields' (D:\Python3.8.8\lib\site-packages\pydantic\fields.py)

**Comment by JavierSanchezCastro** [comment]
Is this a FastAPI problem? If so, which versions of FastAPI and Pydantic are you using?

**Comment by nissansz** [comment]
I don't know which caused the problem.

fastapi                                       0.108.0
pydantic                                      2.5.3
pydantic_core                                 2.14.6

**Reply by JavierSanchezCastro** [reply]
I tried that combination of versions and it works properly, could you show some examples?
